packages/@aws-cdk/aws-events/README.md

```diff
@@ -80,6 +80,7 @@ interface for various AWS services.
 
 The following targets are supported:
 
+* `targets.AwsApi`: Invoke an AWS Lambda function that makes API calls
```

Comment:
Move to the end of the list. Also, the fact that this uses a Lambda function is an implementation detail.